msi/ms7e06/README.md

```diff
@@ -0,0 +1,19 @@
+# MSI MS-7E06 blobs
+
+The directory contains blobs to produce a full Dasharo firmware image
+compatible with MSI PRO Z790-P variants (DDR4 and DDR5 with or without WIFI).
+
+* [me.bin](me.bin) - Intel Management Engine v16.1.30.2255, subject to
+  [INTEL SOFTWARE LICENSE AGREEMENT](../../licenses/pv%20intel%20obl%20software%20license%20agreement%2011.2.2017.pdf)
+* [descriptor.bin](descriptor.bin) - Intel Flash Descriptor, subject to
+  [INTEL SOFTWARE LICENSE AGREEMENT](../../licenses/pv%20intel%20obl%20software%20license%20agreement%2011.2.2017.pdf)
+
+Intel Management Engine and Flash Descriptor have been generated from Intel
+CSME for Raptor Lake-S v16.1.30.2255v5 Consumer software kit using genuine
+components provided by Intel, subject to the [INTEL SOFTWARE LICENSE AGREEMENT](../../licenses/pv%20intel%20obl%20software%20license%20agreement%2011.2.2017.pdf).
+Apart from the components in the CSME software kit, a Chipset Intialization
+binary has been generated from the Intel PCH Chipset Initialization Kit
+v160.02.99.1014, subject to the [INTEL SOFTWARE LICENSE AGREEMENT](../../licenses/pv%20intel%20obl%20software%20license%20agreement%2011.2.2017.pdf).
+
+By using above components you agree to the
```

Comment:
@miczyg1 Please consider looking at my commits from: https://github.com/Dasharo/dasharo-blobs/commits/licenses

Maybe we can have a single disclaimer in a main README

Reply:
I still left the disclaimer in the subdirectories